src/lib/viewers/BaseViewer.js

```diff
@@ -536,6 +536,9 @@ class BaseViewer extends EventEmitter {
      */
     toggleFullscreen() {
         fullscreen.toggle(this.containerEl);
+        if (this.options.showAnnotationsControls) {
```

Comment:
There are several different ways to enter fullscreen mode, so we'll probably want to handle this from within the component. Take a look at MediaControls for an example.